openpype/hosts/maya/api/plugin.py

```diff
@@ -292,6 +295,9 @@ def update_instances(self, update_list):
     def remove_instances(self, instances):
         return self._default_remove_instances(instances)
 
+    def get_pre_create_attr_defs(self):
+        pass
+
```

Comment:
@iLLiCiTiT should this maybe be in the `HiddenCreator` class since you wont need pre-create attributes when the creator is hidden?

Reply:
Nit `get_pre_create_attr_defs` is only on `Creator` class. `HiddenCreator` does not have it... Did it crash for you somewhere?

Reply:
This is the traceback from loading in a `rig`:

```
Traceback (most recent call last):
  File "C:\Users\tokejepsen\OpenPype\openpype\tools\loader\widgets.py", line 1592, in _load_representations_by_loader
    load_with_repre_context(
  File "C:\Users\tokejepsen\OpenPype\openpype\pipeline\load\utils.py", line 323, in load_with_repre_context
    return loader.load(repre_context, name, namespace, options)
  File "C:\Users\tokejepsen\OpenPype\openpype\hosts\maya\api\plugin.py", line 597, in load
    self.process_reference(
  File "C:\Users\tokejepsen\OpenPype\openpype\hosts\maya\plugins\load\load_reference.py", line 205, in process_reference
    self._post_process_rig(name, namespace, context, options)
  File "C:\Users\tokejepsen\OpenPype\openpype\hosts\maya\plugins\load\load_reference.py", line 227, in _post_process_rig
    create_rig_animation_instance(
  File "C:\Users\tokejepsen\OpenPype\openpype\hosts\maya\api\lib.py", line 4149, in create_rig_animation_instance
    create_context.create(
  File "C:\Users\tokejepsen\OpenPype\openpype\pipeline\create\context.py", line 1982, in create
    precreate_attr_defs = creator.get_pre_create_attr_defs() or []
AttributeError: 'CreateAnimation' object has no attribute 'get_pre_create_attr_defs'
```

Reply:
Will propose a fix for line 1982 in `openpype\pipeline\create\context.py`.